app/views/expenditures/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render partial: 'shared/nav', locals: {title:'TRANSACTION', symbol:'←', url: :back}%>
+<p style="color: green"><%= notice %></p>
+<ul class="expense-ul">
+  <% @expenditures.each do |expense| %>
+    <li class="expenditure-items">
+      <strong>Name:</strong> <%= expense.name %>
+      <br>
+      <strong>Amount:</strong> <%= expense.amount %>
+    </li>
+  <% end %>
+</ul>
+
+
+
+<div class="container-make-category">
+<%= link_to 'Create Expense', new_category_expenditure_path(@category), class: "make-category"  %>
+</div>
```

Comment:
- [ ] Please make sure the transitions are presented, and ordered by the most recent as requested in the project requirements. Currently, presented, and ordered by the most last.

![image](https://github.com/quavo19/budget-app/assets/83803414/1d139793-3c30-48fe-b03e-46a5dd0c8532)

Reply:
alright sir, all will be done